mautic/mautic
# [UX] Change stop engaging after a closing the focus to remove the negative wording when creating focus item

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
When creating a focus item, you have this option to decide whether the item should try again with the same user, at another time, if it was closed the first time.
This PR makes the question positive.

Before: `Stop engaging after a closing the focus?`
After: `Engage again if dismissed?`



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Channels > Focus items > New > Builder
3. Select the type "Display a notice" and see the new label

![image](https://github.com/mautic/mautic/assets/116097999/7325f7f6-4bb2-4cab-ae76-8e7953f02ffa)
